packages/mds-compliance-engine/tests/engine/helpers.spec.ts

```diff
@@ -0,0 +1,250 @@
+import { Settings } from 'luxon'
+import { assert } from 'unit.js'
+import { isRuleActive } from '../../engine/helpers'
+
+describe('Tests isRuleActive checking', () => {
+  const oldTimezone = process.env.TIMEZONE
+  before(() => {
+    process.env.TIMEZONE = 'UTC'
+  })
+
+  after(() => {
+    process.env.TIMEZONE = oldTimezone // make sure to reset TIMEZONE to whatever it was before
+    Settings.now = () => new Date().valueOf() // make sure to reset luxon
+  })
+
+  describe('Null everything', () => {
+    it('Tests for rule with no start_time, end_time, or days to be true', () => {
+      assert.deepEqual(
+        isRuleActive({
+          days: null,
+          start_time: null,
+          end_time: null
+        }),
+        true
+      )
+    })
+  })
+
+  describe('Test only days', () => {
+    it('Tests for rule with only days and no start_time/end_time to be true if day aligns', () => {
+      Settings.now = () => new Date('2021-09-15T05:00:00.000Z').valueOf() // this is a wednesday
+
+      assert.deepEqual(
+        isRuleActive({
+          days: ['wed'],
+          start_time: null,
+          end_time: null
+        }),
+        true
+      )
+    })
+
+    it('Tests for rule with only days and no start_time/end_time to be false if day does not align', () => {
+      Settings.now = () => new Date('2021-09-15T05:00:00.000Z').valueOf() // this is a wednesday
+
+      assert.deepEqual(
+        isRuleActive({
+          days: ['mon'],
+          start_time: null,
+          end_time: null
+        }),
+        false
+      )
+    })
+  })
+
+  describe('Tests only start_time && end_time', () => {
+    it('Tests start_time/end_time to be true if current time aligns', () => {
+      Settings.now = () => new Date('2021-09-15T05:00:00.000Z').valueOf()
+
+      assert.deepEqual(
+        isRuleActive({
+          days: null,
+          start_time: '02:00:00',
+          end_time: '10:00:00'
+        }),
+        true
+      )
+    })
+
+    it('Tests start_time/end_time to be false if current time does not align', () => {
+      Settings.now = () => new Date('2021-09-15T05:00:00.000Z').valueOf()
+
+      assert.deepEqual(
+        isRuleActive({
+          days: null,
+          start_time: '09:00:00',
+          end_time: '10:00:00'
+        }),
+        false
+      )
+    })
+
+    describe('Tests edge case: start_time > end_time (across midnight). e.g. 7pm-5am', () => {
+      it('Tests before midnight (true)', () => {
```

Comment:
May want to add two test cases for current time past midnight (early morning), e.g.:

1. current time is 04:00:00 (should still be active since start_date >= end_date)
2. current time is 06:00:00 (should no longer be active, past end_time)